parser-typechecker/src/Unison/FileParser.hs

```diff
@@ -68,7 +68,9 @@ file = do
     let (terms, watches) = (reverse termsr, reverse watchesr)
     -- suffixified local term bindings shadow any same-named thing from the outer codebase scope
     -- example: `foo.bar` in local file scope will shadow `foo.bar` and `bar` in codebase scope
-    let (curNames, resolveLocals) = (Names.deleteTerms0 locals (Names.currentNames names), resolveLocals)
+    let (curNames, resolveLocals) =
+          ( Names.shadowSuffixedTerms0 locals (Names.currentNames names)
+          , resolveLocals )
```

Comment:
Renamed `Names3.deleteTerms0` to `shadowSuffixedTerms0`.

This name resolution stuff was very subtle so I tweaked some stuff like this to improve clarity a bit.